src/mavedb/view_models/score_set.py

```diff
@@ -34,6 +34,10 @@
 from mavedb.view_models.variant import VariantInDbBase
 
 
+class ExternalLink(BaseModel):
+    url: Optional[str]
```

Comment:
Do we want to allow these to be None? For this link it seems like it's the difference between
```
{}
```
and
```
{"ucscGenomeBrowser": {"url": None}}
```
To me, the former seems simpler if we are going to add more links since it would reduce clutter in these objects.

Lmk if you think the other way is better though, totally open to it.

Reply:
I left `url` optional in case we want to support other links that are specified differently — with an ID or some combination of parameters rather than a plain URL. I figured I'd prevent the front end from assuming there's a URL from the beginning, while avoiding breaking API changes. But I do see both sides of the argument here.

If there's no UCSC Genome Browser link, I currently expect to find an empty object (`{}`, your first option). I agree, there should be no occasion to have the second option.

Reply:
Thanks, that makes sense.